src/main/java/com/educare/unitylend/controller/BorrowRequestController.java

```diff
@@ -0,0 +1,65 @@
+package com.educare.unitylend.controller;
+
+import com.educare.unitylend.Exception.ControllerException;
+import com.educare.unitylend.model.BorrowRequest;
+import com.educare.unitylend.service.BorrowRequestService;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Slf4j
+@AllArgsConstructor
+@RestController
+@RequestMapping("/borrow-request")
+public class BorrowRequestController extends BaseController{
+    BorrowRequestService borrowRequestService;
+    @GetMapping("/{userId}")
+    public ResponseEntity<List<String>> getAllCommunities(@PathVariable String userId) throws ControllerException {
```

Comment:
The function name should be `getCommunitiesForUser()`

Reply:
There should be a method for getting the borrowRequest for a given UserId, Community. 

And one for Administration purpose to get all the Borrowrequests.